src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -451,6 +451,7 @@ public static class ObservedData
          public static readonly string MOLECULARWEIGHT = "Molecular Weight";
          public static readonly string SPECIES = "Species";
          public static readonly string STUDY_ID = "Study Id";
+         public static readonly string SUBJECT_ID = "Subject Id";
          public static readonly string GROUP_ID = "Group Id";
          public static readonly string ROUTE = "Route";
```

Comment:
Should we remove this entry then?

Reply:
Why? @AndreDlm requested "Subject Id" to be added. @PavelBal said should not be "Subject Id" but "Patient Id" and @Yuri05 replied "Subject Id" is a generalization of "Patient Id" so it should be as in code now, right?

Reply:
Exactly. We do not need patientid and subject id because one is the generalization of the other. Don't you think?